modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,615 @@
+
+[id="web-console-new-features-1-16_{context}"]
+=== Web console
+
+* Before this release, when using the web console, you could not see the timestamp for the logs that {pipelines-shortname} created. With this update, the web console includes timestamps for all {pipelines-shortname} logs.
+
+* With this update, the pipeline run and task run *list* pages in the web console now have a filter for the data source, such as `k8s` and `TektonResults API`.
+
+* Before this update, when using the web console in the *Developer* perspective, you could not specify the timeout for pipeline runs. With this update, you can set a timeout while starting the pipeline run in the *Developer* perspective of the web console. 
+
+* Before this update, the *Overview* pipeline dashboard only appeared when {tekton-results} was enabled. All the statistics came from only the Results API. With this update, the *Overview* pipeline dashboard is available regardless of whether {tekton-results} is enabled or not. When {tekton-results} is disabled, you can use the dashboard to see the statistics for objects in the cluster.
+
+* With this update, the sample pipelines displayed in the web console use the `v1` version of the {pipelines-shortname} API.
+
+[id="cli-new-features-1-16_{context}"]
+=== CLI
+
+* With this update, you can use the `tkn customrun delete <custom_run_names>` command to delete one or more custom runs.
+
+* With this update, when you run a `tkn <resource> list` command with the `-o` YAML flag, the listed resources are now separated with `---` separators to enhance readability of the output.
+
+[id="pac-new-features-1-16_{context}"]
+=== {pac}
+
+* With this update, if you create two `PipelineRun` definitions with the same name, {pac} logs an error and does not run either of these pipeline runs.
+
+* With this update, the {pac} `pipelines_as_code_pipelinerun_count` metric allows filtering of the `PipelineRun` count by repository or namespace.
+
+* With this update, the `readOnlyRootFilesystem` security context for the {pac} controller, webhook, and watcher is set to `true` by default to increase security and avoid being flagged by the security scanner.
+
+[id="tekton-chains-new-features-1-16_{context}"]
+=== {tekton-chains}
+
+* With this update, when using `docdb` storage in {tekton-chains}, you can configure the `MONGO_SERVER_URL` value directly in the `TektonConfig` CR as the `storage.docdb.mongo-server-url` setting. Alternatively, you can provide this value using a secret and set the `storage.docdb.mongo-server-url-dir` setting to the directory where the `MONGO_SERVER_URL` file is located.
++
+.Example of creating a secret with the `MONGO_SERVER_URL` value
+[source,terminal]
+----
+$ oc create secret generic mongo-url -n tekton-chains \ #
```

Comment:
I don't think this breaks anything as-is, but wondering about the `#` on the end after the line-break, since there's no callout:
```suggestion
$ oc create secret generic mongo-url -n tekton-chains \
```